source/install/cloud-changelog.md

```diff
@@ -3,12 +3,58 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-08-11](#release-2021-08-11)
 - [Release 2021-07-29](#release-2021-07-29)
 - [Release 2021-07-15](#release-2021-07-15)
 - [Release 2021-07-01](#release-2021-07-01)
 - [Release 2021-06-16](#release-2021-06-16)
 - [Release 2021-06-02](#release-2021-06-02)
-- [Release 2021-05-21](#release-2021-05-21)
+
+## Release 2021-08-11
+
+### Highlights
+
+#### Added a global product launcher
+ - 
+
+#### Beta features promoted from Beta to General Availability
+   - Archived channels
+   - Compliance exports
+   - Custom terms of service
+   - Guest Accounts
+   - System Roles
+   - Plugins
+
+#### Focalboard
```

Comment:
Chen, now that Focalboard is released to Cloud, do we want to mention it in the Cloud changelog?

Reply:
cc @chenilim

Reply:
I think this note is fine for now. We are currently in the middle of the rollout, and the plan of record is to roll out to 100% of cloud instances by Aug 16.